src/docs/patterns/account-details.mdx

```diff
@@ -0,0 +1,140 @@
+
+This approach is most effective if you have more than a few account-related actions, or if you want to present more detailed information about a user. The user menu is a good option if you want to avoid overloading the navigation drawer. This technique can also be used in conjunction with a navigation rail.
+
+On mobile, the user menu content should be presented in a [Bottom Sheet](/patterns/overlay#bottom-sheet) instead of a dropdown menu.
+
+<React.Fragment>
+    <ImageGrid
+        images={[MobileUserMenu]}
+        caption={`On mobile-sized screens, the menu contents should be presented in a bottom sheet.`}
+    />
+    <Divider />
+</React.Fragment>
+
+# Contents
+
+Whether you are using the drawer or a dedicated user menu component, the content should be very similar. Most account functions can be located on the account settings page, but you should provide access to the most common / frequent actions in the drawer or user menu.
+
+You should include:
+
+-   A link to the account settings page
+-   A Log Out button
+-   Links to terms of service / privacy policy / license information (if applicable)
+
+When using the user menu, you will likely have more room available if you wish to include other frequently accessed account functions, such as change password, contact support, etc.
+
+<React.Fragment>
+    <Divider />
+</React.Fragment>
+
+# Design Specifications
+
+## Mobile
+
+<ImageGrid images={[Specs1, Specs3, Specs4]} regularWidth />
+
+## Desktop and Tablet
+
+If certain dimensions are not specified, refer to the dimensions suggested in the mobile section above.
+
+<ImageGrid images={[Specs2]} regularWidth />
+<Divider />
+
+# Developers
+
+Use the following components to implement this pattern:
+
+**Angular** and **Ionic**:
+
+-   Angular Material
+    -   [Side Nav](https://material.angular.io/components/sidenav/overview)
+-   @pxblue/angular-components
+    -   [Drawer](https://pxblue-components.github.io/angular/?path=/info/components-drawer--readme)
+    -   [User Menu](https://pxblue-components.github.io/angular/?path=/info/components-user-menu--readme)
+
+**React**:
+
+-   Material UI
+    -   [Drawer](https://material-ui.com/components/drawers/)
+-   @pxblue/react-components
+    -   [User Menu](https://pxblue-components.github.io/react/?path=/info/components-user-menu--get-read-me-story)
+
+**React Native**:
+
+-   @pxblue/react-native-components
+    -   [User Menu](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--user-menu)
```

Comment:
You linked drawer component for Angular and React, but not React Native?

And you linked the PX Blue Angular Drawer, but not the PX Blue React Drawer?